op-guide/pd-control.md

```diff
@@ -0,0 +1,206 @@
+# PD Control instructions
+
+As a command line tool of PD, PD Control is used for obtaining the cluster state information and tuning the cluster.
+
+## Source code compiling
+
+1. [*Go*](https://golang.org/) Version 1.7 or later
+2. In the PD root directory, use `make` command to compile and generate bin/pd-ctl
+
+## Usage
+
+single-command mode:
+
+    ./pd-ctl store -d -u 127.0.0.1:2379
+
+interactive mode:
+
+    ./pd-ctl -u 127.0.0.1:2379
+
+use environment variables:
+
+```bash
+export PD_ADDR=http://127.0.0.1:2379
+./pd-ctl
+```
+
+## command line flags
+
+### \-\-pd,-u
+
++ PD address
++ Default address: http://127.0.0.1:2379
++ Enviroment variable: PD_ADDR
+
+### \-\-detach,-d
+
++ Use single command line mode (not entering readline)
++ Default value: false
+
+## command (command)
+
+### store [delete] \<store_id\>
+
+This is used for displaying the store information or deleting the specified store.
```

Comment:
I think we can remove all `This is used for`, using `Display the store...` is enough.